### PR TITLE
upgrade from Java 11 to Java 17

### PR DESCRIPTION
**Pull request by asolntsev**
FINALLY!!!

### CODE REVIEW

**Review by BorisOsipov — APPROVED** [review]

File: gradle/compilation.gradle [hunk 1]
```diff
@@ -6,12 +6,11 @@ subprojects {
   tasks.withType(JavaCompile).configureEach {
     options.encoding = encoding
     options.debug = true
-    options.release = 11
     sourceCompatibility = 17
```
Comment (rosolko) [inline gradle/compilation.gradle:9]: I think here we don't really need this line anymore

File: gradle/compilation.gradle [hunk 2]
```diff
@@ -6,12 +6,11 @@ subprojects {
...
     options.debugOptions.debugLevel = "source,lines,vars"
   }
 
-  sourceCompatibility = '11'
+  sourceCompatibility = '17'
```
Comment (rosolko) [inline gradle/compilation.gradle:13]:
```suggestion
  sourceCompatibility = targetCompatibility = 17
```

**Review by rosolko — APPROVED** [review]

**Comment by vladimirjegorov** [comment]
A bit unexpected, thought you'd be targeting Java 11, to be aligned with Selenium... but I guess Java 17 is the new reality with Selenide 7.0

**Comment by asolntsev** [comment]
@vladimirjegorov Yes, a bit unexpected. At some moment, we decided to be rebels in QA world. :)

**Pull request merged**